x-pack/legacy/plugins/apm/scripts/upload-telemetry-data/download-telemetry-mapping.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import Ocktokit from '@octokit/rest';
+
+export async function downloadTelemetryMapping(ocktokit: Ocktokit) {
```

Comment:
ATM, the xpack-phone-home template gets updated every time mappings need to be added to the current telemetry service.

Reply:
Resolving this as it's related to the script we discussed.